compiler/src/iree/compiler/Dialect/HAL/Conversion/StreamToHAL/Patterns.cpp

```diff
@@ -332,26 +332,19 @@ struct ResourceAllocOpPattern
         lookupAllocatorAndQueueAffinityFor(allocOp, rewriter);
     auto bufferType = rewriter.getType<IREE::HAL::BufferType>();
 
-    SmallVector<Value> results;
-    for (auto [resourceResult, storageSize] :
-         llvm::zip_equal(allocOp.getResults(), allocOp.getStorageSizes())) {
```

Comment:
Just curious how did we refactor this to be a single result rather than 2?

Reply:
before it could be any number of values - it was intended to be shorthand for multiple allocations but wasn't ever used and just made the op hard to work with - have been meaning to clean it up since I originally wrote it :P